src/test/kotlin/io/spine/protobuf/MessageExtsSpec.kt

```diff
@@ -63,9 +64,8 @@ internal class MessageExtsSpec {
         value.ensureUnpacked() shouldBe value
     }
 
-    @Nested
-    @DisplayName("verify that")
-    internal inner class VerifyThat {
+    @Nested inner class
+    `verify that` {
```

Comment:
Did you intend to test your new API extensions? I am not sure, but it does not look like you did.